studio/data/service-status/edge-functions-status-query.ts

```diff
@@ -0,0 +1,38 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+
+import { get } from 'lib/common/fetch'
+import { ResponseError } from 'types'
+import { serviceStatusKeys } from './keys'
+
+export type EdgeFunctionServiceStatusVariables = {
+  projectRef?: string
+}
+
+export async function getEdgeFunctionServiceStatus(
+  { projectRef }: EdgeFunctionServiceStatusVariables,
+  signal?: AbortSignal
+) {
+  const res = await get(`https://obuldanrptloktxcffvn.supabase.co/functions/v1/health-check`, {
```

Comment:
Should this be a static URL? Or should `projectRef` be interpolated?

Reply:
static URL! but good point - shouldn't need project ref here
imma keep it in the query key though, although we'd be pinging the same endpoint, but its to have RQ check once more each time the user switches projects

this check for edge functions is kinda like a v1 until we're able to make it more fine grained